pkg/logutils/log.go

```diff
@@ -0,0 +1,30 @@
+package logutils
+
+type Log interface {
+	Fatalf(format string, args ...interface{})
+	Errorf(format string, args ...interface{})
+	Warnf(format string, args ...interface{})
+	Infof(format string, args ...interface{})
+
+	Child(name string) Log
+	SetLevel(level LogLevel)
+}
+
+type LogLevel int
+
+const (
+	// debug message, write to debug logs only by logutils.Debug
```

Comment:
comment on exported const LogLevelDebug should be of the form "LogLevelDebug ..."